.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please, remove `.idea` files and add `.idea` to `.gitignore`

Reply:
Done, I will not add `node_modules` for now to `.gitignore`, because not needed here, will never appear that folder.